### PR TITLE
Enable setting environment variables in .ko.yaml

### PR DESCRIPTION
**Pull request by halvards**
Matches the GoReleaser format.

Related: #340

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +146,8 @@ builds:
   - -X main.version={{.Env.VERSION}}
 - id: bar
   main: ./foobar/bar/main.go
+  env:
+  - CGO_ENABLED=1
```
Comment (imjasonh) [inline README.md:150]: We don't tend to support cgo well, can we use another example here instead?

Comment (halvards) [reply README.md]: Good point, changed to `GOCACHE`.

**Comment by halvards** [comment]
Should I commit the changes in `doc/` too? The only changes are the dates in the `Auto generated` comments on each file, there are no changes to the content.

**Review by jonjohnsonjr — DISMISSED** [review]

**Comment by jonjohnsonjr** [comment]
I think this just needs a rebase.

**Comment by halvards** [comment]
Rebased, but Travis CI complains about the s390x build. Other archs seem fine, and the other PR has the same problem.

**Comment by jonjohnsonjr** [comment]
> Rebased, but Travis CI complains about the s390x build. Other archs seem fine, and the other PR has the same problem.

Frustrating :/ just going to merge this anyway but we should probably just drop travis.

**Pull request merged**